wandb/integration/cohere/resolver.py

```diff
@@ -1,25 +1,29 @@
 import logging
-from typing import Any, Dict, List, Optional
+from typing import Any, Dict, List, Optional, Sequence
 
 from wandb.sdk.data_types import trace_tree
-from wandb.sdk.integration_utils.llm import Response
+from wandb.sdk.integration_utils.auto_logging import Response
 
 logger = logging.getLogger(__name__)
 
 
 class CohereRequestResponseResolver:
```

Comment:
Subclass the `ArgumentResponseResolver` here

Reply:
`ArgumentResponseResolver` is a `typing.Protocol`. Protocols are used to express interface contracts, not to provide implementation details. Therefore, you should not typically subclass a protocol to inherit implementation.
Please revert this change.

Reply:
Addressed in: https://github.com/wandb/wandb/pull/5535/commits/ff77a17c16fdb32592a81405d16cb3755b64ca12